01week/datatypes.js

```diff
@@ -0,0 +1,80 @@
+// 1. Write a JavaScript program to display the current day and time.
+
+// Notes: method "Date()" can pull the current date/time
+
+const currentTime = Date();
+console.log(currentTime)
+
+
+// 2. Write a JavaScript program to convert a number to a string.
+
+// Notes: method "String()" can convert numbers to a string.
+
+const value2 = 100;
+const numberToString = String(value2)
+console.log(numberToString)
+
+
+// 3. Write a JavaScript program to convert a string to the number.
+
+// Notes: method "Number()" can convert numbers to a number.
+
+const value1 = "10"
+const stringToNumber =  Number(value1);
+console.log(stringToNumber)
+
+//4. Write a JavaScript program that takes in different datatypes and prints out whether they are a:
+// Boolean
+// Null
+// Undefined
+// Number
+// NaN
+// String
+
+// Notes: "typeof" can tell us the data type
+
+const input = "test"
```

Comment:
This is a  variable. I need a function that accepts ANY argument and returns it's data type